.circleci/config.yml

```diff
@@ -9,7 +9,7 @@ commands:
             - v2-dependencies-{{ checksum "project.clj" }}
             # fallback to using the latest cache if no exact match is found
             - v2-dependencies-
-      - run: lein with-profile -dev,+ci,+test,+refactor-nrepl deps
+      - run: lein with-profile -user,-dev,+ci,+test,+refactor-nrepl deps
```

Comment:
I added `-user` everywhere, lately I've realised that this makes commands more reproducible. It saves time that people can copy-paste command into their terminal, resting assured that ~/.lein won't interfere.

We use this in e.g. all the important https://github.com/clojure-emacs projects.

Reply:
This is a good idea, for all my 'reproducible' commands (`lein test`-aliases and such) I include `-user` too :)